docs/migration_guide.rst

```diff
@@ -1,6 +1,16 @@
 Migration guide
 ===============
 
+**********************
+0.20.0 Migration guide
+**********************
+
+0.20.0 Deprecations
```

Comment:
I would consider categorizing this under `Breaking Changes`. Technically, it's mostly a breaking change on the Starknet side, yet it definitely doesn't classify as a deprecation.